LabATM/Program.cs

```diff
@@ -1,12 +1,41 @@
-﻿using System;
+using System;
 
 namespace LabATM
 {
-    class Program
+  class Program
+  {
+    private static readonly BankAccount account = new BankAccount();
+    static void Main(string[] args)
     {
-        static void Main(string[] args)
+      WriteWelcome();
+      //int PromptForAction();
+
+      static void WriteWelcome()
+      {
+        Console.WriteLine("Welcome to your friendly ATM!\n");
+      }//this is the end of the WriteWelcome()
+
+      static int PromptForAction()
+      {
+        String promptResponse;
```

Comment:
line 20 needs to be set to equal the user's response (via `Console.ReadLine()`), then use line 21 to convert it to an int. Both of these lines should be moved below line 24's `Console.WriteLine("Shall we play a game?  What would you like to do today?\n");` so that it can capture the user's response to that question. 

Also you don't need the `\n` because WriteLine automatically creates a new line for you! :)

Reply:
So, I have the ReadLine and the Convert down on 30 and 31 where I ask on 29 to enter a number....

Reply:
I can't figure out why I have squiggly lines under the PromptForAction()